src/NuGet.Indexing/NuGetQuery.cs

```diff
@@ -194,6 +197,11 @@ private static void AuthorClause(BooleanQuery query, Analyzer analyzer, IEnumera
             query.Add(ConstructClauseQuery(analyzer, "Authors", values), occur);
         }
 
+        private static void PackageTypeClause(BooleanQuery query, Analyzer analyzer, IEnumerable<string> values, Occur occur)
```

Comment:
?

---
In reply to: [75626531](https://github.com/NuGet/NuGet.Services.Metadata/pull/87#discussion_r75626531) [](ancestors = 75626531)